test/jasmine/tests/plotschema_test.js

```diff
@@ -147,6 +147,18 @@ describe('plot schema', function() {
         });
     });
 
+    it('includes xaxis-only items on only the x axis, not y or bare', function() {
+        var items = ['rangeselector', 'rangeslider'];
+
+        var layoutAttrs = plotSchema.layout.layoutAttributes;
+
+        items.forEach(function(item) {
```

Comment:
I'm curious. How did you find out about this? Did you compare `Plotly.PlotSchema.get()` to some base schema output?

Reply:
.... I'm thinking maybe we should start committing a `plot-schema.json` in this repo (possible in `dist/` alongside the bundles). i'm sure some users would find it useful. Thoughts?

Reply:
Ha, can't remember what prompted me to dive into it but it was looking at rangeslider attributes. A committed schema could certainly be useful!